tests/py/multi_gpu_test_case.py

```diff
@@ -0,0 +1,21 @@
+import unittest
```

Comment:
Does this need to be duplicated? can we just use ModelTestCase?

Reply:
More descriptive

Reply:
This part doesnt need to be more descriptive, since its a generic test case framework. the actual test class (TestCompile) needs to be more discriptive